interpreter.py

```diff
@@ -1202,11 +1216,11 @@ def func_configuration_data(self, node, args, kwargs):
     def func_project(self, node, args, kwargs):
         if len(args) < 2:
             raise InvalidArguments('Not enough arguments to project(). Needs at least the project name and one language')
-        if list(kwargs.keys()) != ['subproject_dir'] and len(kwargs) != 0:
-            raise InvalidArguments('project() only accepts the keyword argument "subproject_dir"')
 
         if not self.is_subproject():
             self.build.project_name = args[0]
+        self.active_projectname = args[0]
+        self.build.dep_manifest[args[0]] = kwargs.get('version', 'undefined')
```

Comment:
I'd prefer to use '0' instead of 'undefined'. I think version should be numeric-only